src/lib/Form/Type/Language/LanguageChoiceType.php

```diff
@@ -9,41 +9,19 @@
 namespace EzSystems\EzPlatformAdminUi\Form\Type\Language;
 
 use EzSystems\EzPlatformAdminUi\Form\Type\ChoiceList\Loader\LanguageChoiceLoader;
-use Symfony\Component\Form\AbstractType;
-use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
-use Symfony\Component\OptionsResolver\OptionsResolver;
 
 /**
  * Form Type allowing to select Language.
  *
  * @todo: This can replace LanguageType in the future but it'd require some changes in the frontend as well.
  */
-class LanguageChoiceType extends AbstractType
+class LanguageChoiceType extends AbstractLanguageChoiceType
 {
-    /** @var \EzSystems\EzPlatformAdminUi\Form\Type\ChoiceList\Loader\LanguageChoiceLoader */
-    private $languageChoiceLoader;
-
     /**
      * @param \EzSystems\EzPlatformAdminUi\Form\Type\ChoiceList\Loader\LanguageChoiceLoader $languageChoiceLoader
      */
     public function __construct(LanguageChoiceLoader $languageChoiceLoader)
     {
-        $this->languageChoiceLoader = $languageChoiceLoader;
-    }
-
```

Comment:
What you are doing here is wrong. You don't need `AbstractLanguageChoiceType`. Since you only need to change choice loader in your types, then use `choice_loader` option. Do not create X additional classes to achieve what is already handled well by Symfony.

```php
$formBuilder->add('configured_language_choice_type', LanguageChoiceType::class, [
    'choice_loader' => new ConfiguredLanguagesChoiceLoader(), // or inject from DIC
];
```